src/rdo.rs

```diff
@@ -1114,7 +1114,7 @@ fn inter_frame_rdo_mode_decision<T: Pixel>(
           [MotionVector::default(); 2]
         }
       }
-      PredictionMode::NEAR0MV | PredictionMode::NEAR_NEARMV => {
+      PredictionMode::NEAR0MV | PredictionMode::NEAR_NEAR0MV => {
```

Comment:
Should this be updated to add the new modes?

Reply:
When the new modes are added to RAV1E_INTER_COMPOUND_MODES in #2237, the logic for those modes will be added here.